src/integration/blockchain/ain/services/crypto.service.ts

```diff
@@ -9,30 +9,30 @@ import { Config } from 'src/config/config';
 @Injectable()
 export class CryptoService {
   public verifySignature(message: string, address: string, signature: string): boolean {
-    const blockchains = this.getBlockchainsBasedOn(address);
+    const blockchain = this.getBlockchainsBasedOn(address);
```

Comment:
As stated before, it would probably make sense to add a blockchain parameter here, to include it in the validation.